core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -346,7 +346,7 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
               <collection-summary-tile
                                        collection-id="tile.id"
                                        collection-title="tile.title"
-                                       last-updated-msec="tile.last_updated_msec"
+                                       last-updated-msec="tile.last_updated"
```

Comment:
I looked at the conversation skin directive, and the field is called last_updated_msec, which I also think is a better name. Any reason why we aren't using summary_services.get_displayable_collection_summary_dicts() to retrieve the necessary info?